src/library/components/stem-finder.js

```diff
@@ -182,6 +182,8 @@ const StemFinder = Component({
       searchPage,
       '&interactive_page=',
       searchPage,
+      '&project_page=',
```

Comment:
This probably has to change too after portal changes.

Reply:
Ah, right. Thanks for catching those.